.circleci/config.yml

```diff
@@ -41,12 +44,31 @@ jobs:
       xcode-version: "12.4.0"
     steps:
       - setup_environment
+      - copy_demo_credentials
       - run:
           name: Build App Store Target
           command: bundle exec fastlane build_app_store
 
+  Upload to App Store:
+    executor:
+      name: ios/default
+      xcode-version: "12.4.0"
```

Comment:
nit: wdyt of using a YAML anchor to DRY the `xcode-version:` value that is currently repeated in each job?

On first occurrence in the YAML:
```yaml
executor: &xcode-image
  name: ios/default
  xcode-version: "12.4.0"
```

On all subsequent jobs needing the same executor:
```yaml
executor: *xcode-image
```

Reply:
Good point. I keep thinking about it and never do it. _Later is never_.